tests/mtouch/MiscTests.cs

```diff
@@ -169,6 +169,11 @@ public void PublicSymbols (Profile profile)
 					case "_objc_msgSend_stret":
 					case "_objc_msgSendSuper_stret":
 						return false;
+					// networkable AOT profiler - see https://github.com/mono/mono/pull/14936
+					case "_add_to_fd_set":
+					case "_close_socket_fd":
+					case "_setup_command_server":
```

Comment:
Could these symbols be prefixed somehow? The reason is that we want to try to avoid using symbols that might exist in other third-party libraries, because usually everything is linked together statically on iOS, which makes clashing symbols a very bad idea.

Reply:
I don't think these should be public, they are helper functions used only internally to the log profiler module.

(and now that I took a look, the `_proflog_` prefix whitelist above shouldn't have been necessary too, it only applies to `proflog_parse_args` which afaik isn't part of the public API too. It looks like the module just exposes all functions instead of just ones marked with `MONO_API`.

Reply:
I think we need to loop @radekdoulik into the discussion, as the author of the relevant PR

Reply:
These indeed shouldn't be exposed. They are just helper functions shared between log and aot profiler.

Reply:
It looks like the profiler directory doesn't add `$(SHARED_CFLAGS)` to `AM_CPPFLAGS`, so doesn't pass `-fvisibility=hidden`, so all symbols are public. Am testing to see if it still works, with that changed.

Reply:
https://github.com/mono/mono/pull/16335